gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -1,3 +1,22 @@
+local function PlayerSprint(trySprinting, moveKey)
+	if SERVER then return end
```

Comment:
If this is a clientside function only, why is it even defined on the server?

Reply:
It used to be a local clientside function that's used in a shared hook. That leaves us with four options:
Define a local var in shared realm and populate it with the function in the client realm
Make PlayerSprint global
Split the shared hook into a clientside and a serverside hook
Or do it the way I did it
All have benefits and disadvantages. Ill change it according to your preference.

Reply:
I thought about only defining it on the clientside, as it is secured by another `if CLIENT ...` below, and the function is never called serverside. I think it should be moved back into the previous position, where it was only defined on the client (the big `if SERVER then ... else .... here ... end`. This will also ensure that it is not callable on the server and fail when used otherwise.

Reply:
I think you misunderstood me. It's not only a matter of realm, but also a matter of variable scope. It's a local function and therefore cannot be referenced outside of the block it was defined in. Therefore, putting it back into the else block leads to a nil error, since the shared hook is outside of the else block.

Reply:
I see, then its ok as it is now. 👍🏻